editor.planx.uk/src/ui/editor/RichTextInput/RichTextInput.tsx

```diff
@@ -200,8 +200,8 @@ export const emptyContent = "<p></p>";
 const linkSelectionError = (selectionHtml: string): string | null => {
   if (selectionHtml.startsWith("<p>") && selectionHtml.endsWith("</p>")) {
     const text = selectionHtml.slice(3, -4);
-    const lowercaseText = text.toLowerCase();
-    if (lowercaseText.includes("click") || lowercaseText.includes("here")) {
+    const lowercaseText = text.toLowerCase().trim().replace(/[.,]/g, "");
+    if (lowercaseText == "click here" || lowercaseText == "clicking here") {
```

Comment:
nit: We should use `===` here instead of `==` ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality))